runtime/src/config.c

```diff
@@ -347,7 +347,7 @@ void installConfigVar(const char* varName, const char* value,
   configVar->moduleName = chpl_glom_strings(1, moduleName);
   configVar->defaultValue = chpl_glom_strings(1, value);
   configVar->setValue = NULL;
-  configVar->private = private;
+  configVar->private = isprivate;
```

Comment:
How does `private` as a member name work? I think we should be consistent and use `isPrivate` everywhere.

I just tried a test, and `private` cannot be the name of a member in C++ mode.

Reply:
Yes but this one is fine because it is in a .c file. The header needed to be updated because we include it from C++ compilation in some settings.

Reply:
The C file did not need to be changed, then.

I thought the C file was changed because you were doing it in C++ mode with `-xc++` or similar.

Parameter names in header prototypes need not match the function definition.

Using `isPrivate` (camelCase) everywhere is preferred.

Reply:
I changed the C file because Lydia originally changed the prototype to match.

It would be fine with me to change the uses of `private` in the C file to `isPrivate` but I am not going to do that.